# Exception所有程序异常类的父类
try:
    print(num)
except Exception as result:
    print(result)